mhs/outbound/outbound/transmission/outbound_transmission.py

```diff
@@ -56,14 +57,9 @@ def __init__(self, client_cert: str, client_key: str, ca_certs: str, max_retries
                             "proxy_host": self._proxy_host,
                             "proxy_port": self._proxy_port
                         })
-            # ******************************************************************************************************
-            # TLS CERTIFICATE VALIDATION HAS BEEN TEMPORARILY DISABLED! This is required because Opentest's SDS
-            # instance currently returns endpoints as IP addresses. This MUST be changed before this code is used in
-            # a production environment
-            # ******************************************************************************************************
             response = await CommonHttps.make_request(url=url, method="POST", headers=headers, body=message,
                                                       client_cert=self._client_cert, client_key=self._client_key,
-                                                      ca_certs=self._ca_certs, validate_cert=False,
+                                                      ca_certs=self._ca_certs, validate_cert=self._validate_cert,
```

Comment:
why not set that in common_https.py 'make_request' method?
There we have a default validate_cert: bool = True